plugin/api.go

```diff
@@ -932,6 +932,12 @@ type API interface {
 	//
 	// Minimum server version: 5.18
 	PluginHTTP(request *http.Request) *http.Response
+
+	// PublishUserTyping publishes a user is typing WebSocket event.
```

Comment:
The documentation should make more clear that `parentId` is an optional parameter.